src/SixLabors.Fonts/SystemFonts.cs

```diff
@@ -33,15 +33,16 @@ public static class SystemFonts
         /// Finds the specified font family from the system font store.
         /// </summary>
         /// <param name="fontFamily">The font family.</param>
-        /// <returns>The family if installed otherwise null</returns>
+        /// <returns>The found family.</returns>
```

Comment:
Thanks for doing this. Really appreciate it! Just one tiny typo.  found => font.

Reply:
No problem at all! I actually chose "The found family" over "The font family" as the `TryFind` method below refers to the out-variable as "The found family". Happy to still change it (or both if you like) 🙂

Reply:
Oh... ok, yeah let's leave it as-is for now then.